recipes/libcorrect/all/patches/0002-Export-symbols-for-windows.patch

```diff
@@ -0,0 +1,296 @@
+From 182c276c7334c56629a7181ffe60c8cb0a783ffb Mon Sep 17 00:00:00 2001
+From: Brennan Ashton <bashton@brennanashton.com>
+Date: Wed, 29 Jan 2020 00:03:04 -0800
+Subject: [PATCH 2/2] Export symbols for windows
+
+Signed-off-by: Brennan Ashton <bashton@brennanashton.com>
+---
+ CMakeLists.txt        | 10 ++++++++-
+ include/correct-sse.h | 20 ++++++++++-------
+ include/correct.h     | 49 +++++++++++++++++++++---------------------
+ include/fec_shim.h    | 50 +++++++++++++++++++++----------------------
+ 4 files changed, 71 insertions(+), 58 deletions(-)
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 6169ac4..6bc863d 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -5,6 +5,7 @@ include(CheckIncludeFiles)
+ include(CheckCSourceCompiles)
+ include(CMakePushCheckState)
+ include(CheckCCompilerFlag)
++include(GenerateExportHeader)
```

Comment:
I will steal this information and use it for future projects!
Thanks for the tip!

Reply:
I didn't know about too!